src/DataLayouts/DataLayouts.jl

```diff
@@ -46,6 +46,13 @@ abstract type AbstractData{S} end
 
 Base.size(data::AbstractData, i::Integer) = size(data)[i]
 
+"""
+    universal_size(::AbstractData)
+
+Return sizes `(Ni, Nj, Nf, Nv, Nh)`, for all
+data types. If a dimension does not exist,
+then return 1 for that dimension.
```

Comment:
I know that this is used often in the code, but could you please define the various `(Ni, Nj, Nf, Nv, Nh)` in this docstring too?

Reply:
Yes, will do